README.md

```diff
@@ -11,6 +11,13 @@ cd app
 yarn # to install dependencies
 yarn dev # run the application
 ```
+## Docker
+To run application using Docker, run the following commands from the root folder:
+```bash
+docker build . -f app/Dockerfile # will return 'Successfully built <IMAGE>'
+docker run -dp 3000:3000 <IMAGE>
```

Comment:
I'd encourage people manually running the image to keep the process in the foreground and not to daemonize it
```suggestion
docker run -p 3000:3000 connectivity-intake:latest
```